apps/11gist-blog/script.js

```diff
@@ -1,5 +1,48 @@
 'use strict';
 
 $(document).ready(function() {
-  // You code here
+    // You code here
+    function formatDate(date) {
+        var arrDate = date.split('-');
+        return arrDate[1] + '/' + arrDate[2] + '/'
+        arrDate[0]
+    }
+
+    $.ajax('https://api.github.com/users/sanderson60/gists', {
+        success: function getGists(response) {
+            response.forEach(function(post) {
+                if (post.description.match('^#post')) {
+                    var postDesc = post.description.replace(/#post /g, '');
+                    var postURL = post.url.replace(/http:\/\/127.0.0.1:8080/g, '');
+                    var postDate = formatDate(post.created_at.split('T')[0]);
+                    var str = '<li><a href="#" data-url="' + postURL + '">' + postDesc + '</a> | ' + postDate + '</li>';
+                    console.log(str);
+                    $("#posts").append(str);
+                }
+            })
+            $("#posts a").click(function() {
+                event.preventDefault();
```

Comment:
I had some troubles and tried a few things. You are correct it is wrong and I have updated the file.